src/main/java/seedu/address/logic/commands/HelpCommand.java

```diff
@@ -9,10 +9,16 @@ public class HelpCommand extends Command {
 
     public static final String COMMAND_WORD = "help";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Shows program usage instructions.\n"
-            + "Example: " + COMMAND_WORD;
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Shows program usage instructions.\n";
+
+    public static final String SHOWING_HELP_MESSAGE = "Opened help window.\n"
+        + "add: "+ "Add a contact.\n"
```

Comment:
i think help information shouldn't go into the command results! u might instead want to look at the fxml files.